cmsdb/processes/mttbar.py

```diff
@@ -0,0 +1,1700 @@
+    "hscalar_tt_sl_m_1000_w_25p0_res",
+    "hscalar_tt_sl_m_1000_w_25p0_int",
+    "hpseudo_tt",
+    "hpseudo_tt_sl",
+    "hpseudo_tt_sl_m_365_w_91p25",
+    "hpseudo_tt_sl_m_365_w_91p25_res",
+    "hpseudo_tt_sl_m_365_w_91p25_int",
+    "hpseudo_tt_sl_m_400_w_100p0",
+    "hpseudo_tt_sl_m_400_w_100p0_res",
+    "hpseudo_tt_sl_m_400_w_100p0_int",
+    "hpseudo_tt_sl_m_500_w_125p0",
+    "hpseudo_tt_sl_m_500_w_125p0_res",
+    "hpseudo_tt_sl_m_500_w_125p0_int",
+    "hpseudo_tt_sl_m_600_w_150p0",
+    "hpseudo_tt_sl_m_600_w_150p0_res",
+    "hpseudo_tt_sl_m_600_w_150p0_int",
+    "hpseudo_tt_sl_m_800_w_200p0",
+    "hpseudo_tt_sl_m_800_w_200p0_res",
+    "hpseudo_tt_sl_m_800_w_200p0_int",
+    "hpseudo_tt_sl_m_1000_w_250p0",
+    "hpseudo_tt_sl_m_1000_w_250p0_res",
+    "hpseudo_tt_sl_m_1000_w_250p0_int",
+    "hpseudo_tt_sl_m_365_w_36p5",
+    "hpseudo_tt_sl_m_365_w_36p5_res",
+    "hpseudo_tt_sl_m_365_w_36p5_int",
+    "hpseudo_tt_sl_m_400_w_40p0",
+    "hpseudo_tt_sl_m_400_w_40p0_res",
+    "hpseudo_tt_sl_m_400_w_40p0_int",
+    "hpseudo_tt_sl_m_500_w_50p0",
+    "hpseudo_tt_sl_m_500_w_50p0_res",
+    "hpseudo_tt_sl_m_500_w_50p0_int",
+    "hpseudo_tt_sl_m_600_w_60p0",
+    "hpseudo_tt_sl_m_600_w_60p0_res",
+    "hpseudo_tt_sl_m_600_w_60p0_int",
+    "hpseudo_tt_sl_m_800_w_80p0",
+    "hpseudo_tt_sl_m_800_w_80p0_res",
+    "hpseudo_tt_sl_m_800_w_80p0_int",
+    "hpseudo_tt_sl_m_1000_w_100p0",
+    "hpseudo_tt_sl_m_1000_w_100p0_res",
+    "hpseudo_tt_sl_m_1000_w_100p0_int",
+    "hpseudo_tt_sl_m_365_w_9p125",
+    "hpseudo_tt_sl_m_365_w_9p125_res",
+    "hpseudo_tt_sl_m_365_w_9p125_int",
+    "hpseudo_tt_sl_m_400_w_10p0",
+    "hpseudo_tt_sl_m_400_w_10p0_res",
+    "hpseudo_tt_sl_m_400_w_10p0_int",
+    "hpseudo_tt_sl_m_500_w_12p5",
+    "hpseudo_tt_sl_m_500_w_12p5_res",
+    "hpseudo_tt_sl_m_500_w_12p5_int",
+    "hpseudo_tt_sl_m_600_w_15p0",
+    "hpseudo_tt_sl_m_600_w_15p0_res",
+    "hpseudo_tt_sl_m_600_w_15p0_int",
+    "hpseudo_tt_sl_m_800_w_20p0",
+    "hpseudo_tt_sl_m_800_w_20p0_res",
+    "hpseudo_tt_sl_m_800_w_20p0_int",
+    "hpseudo_tt_sl_m_1000_w_25p0",
+    "hpseudo_tt_sl_m_1000_w_25p0_res",
+    "hpseudo_tt_sl_m_1000_w_25p0_int",
+    "rsgluon_tt",
+    "rsgluon_tt_m_500",
+    "rsgluon_tt_m_1000",
+    "rsgluon_tt_m_1500",
+    "rsgluon_tt_m_2000",
+    "rsgluon_tt_m_2500",
+    "rsgluon_tt_m_3000",
+    "rsgluon_tt_m_3500",
+    "rsgluon_tt_m_4000",
+    "rsgluon_tt_m_4500",
+    "rsgluon_tt_m_5000",
+    "rsgluon_tt_m_5500",
+    "rsgluon_tt_m_6000",
+]
+
+
+from order import Process
+from scinum import Number
+
+#
+# Z prime
+#
+
+zprime = Process(
+    name="zprime",
+    label=r"Z'",
+    id=60000,
+    xsecs={13: Number(0.1)},  # TODO
+)
+
+
+zprime_tt = zprime.add_process(
+    name="zprime_tt",
+    label=rf"{zprime.label} $\rightarrow t\bar{{t}}$",
+    id=61000,
+    xsecs={13: Number(0.1)},  # TODO
+)
+
+# width / mass = 10%
+
+zprime_tt_m_400_w_40 = zprime_tt.add_process(
+    name="zprime_tt_m_400_w_40",
```

Comment:
If you agree with the naming scheme in the above comment, we could rename this likewise to `zprime_tt_m400_w40`. The same would apply to the process variable names and links of processes into the datasets added above 😇

Reply:
OK